osquery/tables/system/hash.cpp

```diff
@@ -232,13 +238,19 @@ void genHashForFile(const std::string& path,
   r["sha256"] = std::move(hashes.sha256);
 
   if (isPlatform(PlatformType::TYPE_POSIX) && context.isColumnUsed("ssdeep")) {
-    r["ssdeep"] = genSsdeepForFile(path);
+    auto status = genSsdeepForFile(path, r["ssdeep"]);
+
+    if (!status.ok()) {
+      logger.log(google::GLOG_WARNING, status.getMessage());
+    }
   }
 
   if (FLAGS_disable_hash_cache) {
     context.setCache(path, tr);
   }
 
+  r["pid_with_namespace"] = "0";
```

Comment:
That is to avoid osquery spitting out an "error" when in verbose mode, since the column is an INTEGER and if it's empty it fails to cast it to an integer, changing it then to 0.